content/api/vision/coordinates.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Coordinates
+---
+
+[Polar coordinates](https://en.wikipedia.org/wiki/Polar_coordinate_system) are represented by a distance from a set point and an angle from a set direction. [Cartesian coordinates](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) are ones that are defined using a unique set of values (typically _x_, _y_ and _z_) to specify a point in space.
+
+{{% notice note %}}
+All distances are measured in metres.
+{{% /notice %}}
+
+## Polar Coordinates
+The polar coordinates are taken from the position of the webcam.
+
+ - `distance_metres` - the distance from the camera to the marker.
+ - `rot_x_rad` / `rot_x_deg` - the angle of rotation in radians/degrees about the x axis. The x axis runs horizontally in front of the camera. A positive angle would be one where the marker is in front of the camera.
```

Comment:
This second sentence seems odd - surely all markers seen by the camera must be in front of it? Perhaps a sentence like the corresponding one for the y axis would make more sense here.

Reply:
I'm not entirely sure how we'd explain this. The chances of them actually needing the `x` variant are fairly low besides getting on-ground distance. I think it's likely both need a rewrite, however I don't know what the correct wording is.

Reply:
I think the issue comes from us trying to define only part of the axis -- the location of the positive direction, but not the location of zero degrees.

How about saying "Angles are measured from directly above the robot around in front of the robot. Any markers the camera can see will be in the range 0° to 180°, with 90° meaning the marker is directly in front of the camera."

I'm not sure about the "directly in front of the camera" bit as it implies that markers off to the side would have a different value. However, I also don't want to say "at the same level as the camera" since that would be wrong if the camera is pointed up or down at all.

Thoughts?

Reply:
I like the looks of that, however I think it's worth changing the bounds, as i'm almost 100% sure the camera does not have a 180 degree FoV, which might lead to confusion